hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/util/LogAdapter.java

```diff
@@ -17,62 +17,34 @@
  */
 package org.apache.hadoop.util;
 
-import org.apache.commons.logging.Log;
 import org.slf4j.Logger;
 
-class LogAdapter {
-  private Log LOG;
-  private Logger LOGGER;
+@Deprecated
+final class LogAdapter {
```

Comment:
I wonder what the purpose of this class. Looks like it can be simply replaced with using Logger directly, I think.
maybe it can removed later.

Reply:
Sure, will have a sub-task for this too

Reply:
There is no IA annotation for this class and it is package private, I think here we could just remove it if we can remove all the references?